dags.yaml

```diff
@@ -116,6 +116,10 @@ bqetl_subplat:
     operational DB as well as derived tables based on that data.
 
     Depends on `bqetl_fxa_events`, so is scheduled to run a bit after that.
+
+    Stripe data for stripe_external__itemized_payout_reconciliation__v5 has
```

Comment:
I'd maybe just include a note that "for this reason we expect this task to fail frequently and will continue to retry until the data becomes available in Stripe".

I'd also maybe include the expected error message so that the person looking at it knows it's not a different issue.

```
Error: Request req_OTssZ0Zv1cEmmm: Data for the report type payout_reconciliation.itemized.5 is only available through 2022-05-08 12:00:00 UTC; you requested `interval_end` = 2022-05-09 00:00:00 UTC.
```